front/components/vaults/VaultDataSourceViewContentList.tsx

```diff
@@ -320,31 +312,7 @@ export const VaultDataSourceViewContentList = ({
     ]
   );
 
-  const emptyVaultContent =
-    isManaged(dataSourceView.dataSource) && vault.kind !== "system" ? (
-      isAdmin ? (
-        <Button
-          label="Manage Data"
-          icon={Cog6ToothIcon}
-          onClick={() => {
-            if (systemVault) {
-              void router.push(
-                `/w/${owner.sId}/vaults/${systemVault.sId}/categories/${dataSourceView.category}`
-              );
-            }
-          }}
-        />
-      ) : (
-        <RequestDataSourceModal
-          dataSources={[dataSourceView.dataSource]}
-          owner={owner}
-        />
```

Comment:
So we getting rid of this the whole request/manage? Fine with me but iirc this was wanted 🤔

Reply:
Sorry, restored that one .. but still, the `RequestOrAddDataFromDataSourceModal` that was in front/components/DataSourceViewResourceSelectorTree.tsx is not anymore in ContentNodeTree